gadget/install/encrypt.go

```diff
@@ -204,7 +169,7 @@ func cryptsetupAddKey(key EncryptionKey, rkey RecoveryKey, node string) error {
 		return fmt.Errorf("cannot open recovery key pipe: %v", err)
 	}
 	n, err := file.Write(rkey[:])
-	if n != recoveryKeySize {
+	if n != len(rkey) {
```

Comment:
previously this used a constant for the recovery key size and I see that we moved the definition for this constant to another package, secboot/encrypt.go, should that constant be exported and used here? or is it good enough that we use the length as the size of the recovery key?

Reply:
I believe this should be safe since the key is an array of `recoveryKeySize` bytes, and not a slice.

Reply:
Also there's a forthcoming PR in the pipeline that replaces this entire function with a [helper provided by snapcore/secboot](https://github.com/snapcore/secboot/pull/54/files#diff-9662d8c5fc8c415f21e1ecc3f784ab73R565).